batch/kueue-intro/team-a-job.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_batch_kueue_intro_team_a_job]
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: team-a # Job under team-a namespace
+  generateName: sample-job-
+  annotations:
+    kueue.x-k8s.io/queue-name: team-a-lq # Point to the LocalQueue
+spec:
+  ttlSecondsAfterFinished: 60 # Job will be deleted after 60 seconds
+  parallelism: 3 # This Job will have 3 replicas running at the same time
+  completions: 3 # This Job requires 3 completions
+  suspend: true # Set to true and allow Kueue to control the Job
+  template:
+    spec:
+      nodeSelector:
+        cloud.google.com/gke-accelerator: "nvidia-tesla-t4" # Specify the GPU hardware
+      containers:
+      - name: dummy-job
+        image: gcr.io/k8s-staging-perf-tests/sleep:latest
+        args: ["10s"] # Sleep for 10 seconds
+        resources:
+          requests:
+            cpu: "1"
+            memory: "100Mi"
+            ephemeral-storage: "100Mi"
+            nvidia.com/gpu: "1"
```

Comment:
Maybe we can comment this line, to make the demo cheaper to run.

But leaving the comment could serve as a pointer that you can use GPUs.

Reply:
if cost is a concern, we can remove it. But maybe this is a good opportunity to showcase autopilot and GPUs. Since it is also a new feature coming to autopilot (currently in preview). The only concern is the autopilot must use rapid release channel for GPU to work on Autopilot. We can keep an eye on this and adjust accordingly when I have more talks with the TWs

Reply:
@mastersingh24 do we need to set both requests and limits in AP? is requests enough?